pkg/controller/artifacts/entrypoint.go

```diff
@@ -27,25 +27,12 @@ datasources:
 - name: prometheus
   type: prometheus
   access: proxy
-  {{- if eq .DSType "common-service" }}
-  url: https://{{ .PrometheusFullName }}:{{ .PrometheusPort }}
-  {{- end }}
-  {{- if eq .DSType "openshift" }}
   url: http://127.0.0.1:9096
-  {{- end }}
   
   isDefault: true
   jsonData:
     keepCookies:
       - cfc-access-token-cookie
-  {{- if eq .DSType "common-service" }}
```

Comment:
how is the authentication (and proper namespace scoping) done with OCP app monitoring?

Reply:
as per ibm-grafana-ocpthanos-proxy repo(https://github.com/IBM/ibm-grafana-ocpthanos-proxy) readme file

--thanos-token-file The token file passed to OCP thanos-querier service for authentication. Default value: "/var/run/secrets/kubernetes.io/serviceaccount/token"